### PR TITLE
MRG: Fix imsave

### LINKED ISSUE #209: save_image broken
The `save_image` method tries to use the `imsave` function in `scipy.misc`. However, it appears that this function was removed from `scipy` some time ago (around 0.13). The `imsave` function now resides in `matplotlib.pyplot`.

You can recreate this bug with the following:
```
conda create -n mayavi python=3
source activate mayavi
conda install -c conda-forge mayavi scipy matplotlib
conda install ipython pyqt=4.11
pip install pysurfer
```

followed by:
```
ipython
$ %gui qt
$ from surfer import Brain
$ brain = Brain('fsaverage', 'split', ' 'inflated')
$ brain.save_image("./test.pdf")
```
(there's not really a need to do this in `ipython)

### PR DESCRIPTION
**Pull request by larsoner**
Closes #209.

@mwaskom I used your approach but left the figure-generation private for now. Feel free to make a follow-up PR to make it public with proper documentation, etc.

### CODE REVIEW

**Comment by mwaskom** [comment]
Haven't tested but LGTM

File: surfer/viz.py [hunk 1]
```diff
@@ -2652,7 +2652,7 @@ def save_movie(self, fname, time_dilation=4., tmin=None, tmax=None,
             raise ValueError("tmax=%r is greater than the latest time point "
                              "(%r)" % (tmax, self._times[-1]))
         n_frames = floor((tmax - tmin) * time_dilation * framerate)
-        times = np.arange(n_frames)
+        times = np.arange(n_frames).astype(float)
```
Comment (mwaskom) [inline surfer/viz.py]: Probably not a big performance hit here, but you could pass the dtype to arange to avoid making a copy.

**Comment by larsoner** [comment]
Okay CIs are happy (except pending OSX but that usually takes forever). I added a 3.6 CI, fixed related errors, and fixed a couple of bare `except:`s we had. Ready for review/merge from my end.

**Comment by larsoner** [comment]
(actually not sure about AppVeyor yet, either, but it passed last time I let it finish)

**Comment by christianbrodbeck** [comment]
Is there an advantage over using Matplotlib's `imsave` (https://github.com/matplotlib/matplotlib/blob/5f63dd42ec6deec5c85ab975d7d879c7c81cb161/lib/matplotlib/image.py#L1305), which seems to be more efficient for PNG but do the same otherwise?

**Comment by larsoner** [comment]
I assumed it was just an efficiency thing so I didn't include it to reduce complexity, but I don't know for sure

**Comment by christianbrodbeck** [comment]
Oh, I didn't mean to include the special case, but whether it wouldn't be easier to just use `matplotlib.image.imsave`

**Comment by larsoner** [comment]
This moves toward a use case @mwaskom wanted, see his comments toward the end of #209

**Comment by christianbrodbeck** [comment]
Aah, makes sense

**Comment by larsoner** [comment]
Anybody else want to look (or @mwaskom want to look again now that everything passes)? If not I'll merge at the end of the week

**Pull request merged**